src/main/java/duke/UI.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import duke.task.Task;
+
+/**
+ * UI class handles interactions with the user.
+ */
+
+public class UI {
+    public BufferedReader br;
+    public final String LINE = "-----------------------------";
+    public UI() {
+        br = new BufferedReader(new InputStreamReader(System.in));
+    }
+
+    private void line() {
+        System.out.println(LINE);
+    }
+
+    public void startMsg() {
```

Comment:
Similarly for these methods as well. Should be `showStartMsg` and also `showByeMsg`.